posthog/models/action/action.py

```diff
@@ -28,6 +46,7 @@ class Meta:
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
     bytecode: models.JSONField = models.JSONField(null=True, blank=True)
     bytecode_error: models.TextField = models.TextField(blank=True, null=True)
+    steps_json: models.JSONField = models.JSONField(null=True, blank=True)
```

Comment:
It would be pretty great if we could name this field `match_groups`, because that's what it contains. "Steps" have always been misnamed, confusing people into thinking we have sequential matching like "user did A, then B". But using this accurate term requires renaming in other places too, so fine with sticking with good ol `steps` too

Reply:
Hmmm if it wasn't part of the API already then I'd be tempted to do the change but potentially the steps are in use by someone so we would _only_ have it be called match_groups at the model level. Could be more confusing?